Library/Formula/unrar.rb

```diff
@@ -15,6 +15,11 @@ class Unrar < Formula
   def install
     system "make"
     bin.install 'unrar'
+    system "make", "clean"
+
+    system "make", "lib"
+    mv "./libunrar.so", "./libunrar.dylib"
```

Comment:
I'm not sure I understand.  Do you want me to write an email to RARLab about their freeware?  I can do so, but I wouldn't my breath :-)